starbook-backend/flask_methods.py

```diff
@@ -57,7 +65,7 @@ def internal_server_error(error):
 
         @app.before_request
         def verify_token():
-            if DEBUG or request.method == 'OPTIONS':
+            if DEBUG or request.method == 'OPTIONS' or 'api' not in request.path:
```

Comment:
what about random files that contain 'api' in the name

Reply:
changed to another condition where i check if the request is for a static file